test/src/test/java/org/corfudb/integration/SecurityIT.java

```diff
@@ -0,0 +1,220 @@
+package org.corfudb.integration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.reflect.TypeToken;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuError;
+import org.corfudb.util.NodeLocator;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+/**
+ * This test suit exercises the ability to enable TLS on Corfu servers and runtime
+ *
+ * Created by Sam Behnam on 8/13/18.
+ */
+@Slf4j
+public class SecurityIT extends AbstractIT {
+    private String corfuSingleNodeHost;
+    private int corfuStringNodePort;
+    private String singleNodeEndpoint;
+
+    private boolean tlsEnabled;
+    private String serverPathToKeyStore;
+    private String serverPathToKeyStorePassword;
+    private String serverPathToTrustStore;
+    private String serverPathToTrustStorePassword;
+    private String runtimePathToKeyStore;
+    private String runtimePathToKeyStorePass;
+    private String runtimePathToTrustStore;
+    private String runtimePathToTrustStorePass;
+
+    /* A helper method that start a single TLS enabled server and returns a process. */
+    private Process runSinglePersistentServerTls() throws IOException {
+        return new AbstractIT.CorfuServerRunner()
+                .setHost(corfuSingleNodeHost)
+                .setPort(corfuStringNodePort)
+                .setTlsEnabled(tlsEnabled)
+                .setKeyStore(serverPathToKeyStore)
+                .setKeyStorePassword(serverPathToKeyStorePassword)
+                .setTrustStore(serverPathToTrustStore)
+                .setTrustStorePassword(serverPathToTrustStorePassword)
+                .setLogPath(getCorfuServerLogPath(corfuSingleNodeHost, corfuStringNodePort))
+                .setSingle(true)
+                .runServer();
+    }
+
+    /**
+     * This is a helper method that load the properties required for creating corfu servers before
+     * each test by converting them from the values provided in CorfuDB.properties. Note that it
+     * will throw {@link NumberFormatException} if the numerical properties provided in CorfuDB.properties
+     * are not well-formed.
+     */
+    @Before
+    public void loadProperties() {
+        // Load host and port properties
+        corfuSingleNodeHost = PROPERTIES.getProperty("corfuSingleNodeHost");
```

Comment:
Duplicate code.

Reply:
Similar to previous point, although it is a correct observation that for now SecurityIT uses the same values for host and port and hense the code for retrieving them is similar. this similarity is not inherent can change independently for each test. For the I want to keep them specific to each test and not centralized it in AbstractIT.